CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ for further information about branching and tagging conventions.
 - Warning message when showing recovery seed on signup page
 - QR code plugin wrapper
 - Added transaction errors handling
+- Added depositing with CoinpaymentsDepositModule in MovementsTopBarReitModule
+- Added WithdrawalDrawerPseudoModule in MovementsTopBarReitModule
```

Comment:
* use quotes `` for entity names
* don't write the word added, it is already present in the header

Reply:
ok